neurodamus/node.py

```diff
@@ -940,9 +942,13 @@ def _report_build_params(self, rep_name, rep_conf, target, pop_offsets_alias_pop
         )
 
     #
-    def _report_write_coreneuron_config(self, rep_name, rep_conf, target, rep_params):
+    def _report_write_coreneuron_config(self, rep_conf, target, rep_params):
         target_spec = TargetSpec(rep_conf["Target"])
 
+        if SimConfig.restore_coreneuron:
```

Comment:
Can we add a comment here addressing this solution is for the case of no change of reporting in restore.  For feature requiring different reports in restore, the work flow needs to be adapted.